### PR TITLE
Updated documentation for REMREM Publish Split

### PR DESCRIPTION
**Pull request by Jibandeep**
Updated documentation for REMREM Publish Split

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -182,181 +198,34 @@ <h2 id="project_tagline"></h2>
...
 
-[{"id":"9cdd0f68-df85-44b0-88bd-fc4163ac90a0","status_code":200,"result":"SUCCESS","message":"Event sent successfully"},{"id":"9cdd0f68-df85-44b0-88bd-fc4163ac90a1","status_code":200,"result":"SUCCESS","message":"Event sent successfully"}]
+[{"id":"9cdd0f68-df85-44b0-88bd-fc4163ac90a0","statusCode":200,"result":"SUCCESS"},{"id":"9cdd0f68-df85-44b0-88bd-fc4163ac90a1","statusCode":200,"result":"SUCCESS"}]
```
Comment (xmairaa) [inline index.html]: it is not statusCode, Its look like status_code

File: index.html [hunk 2]
```diff
@@ -182,181 +198,34 @@ <h2 id="project_tagline"></h2>
 
 <pre><code>$ curl -H "Content-Type: application/json" -X POST -d '["{"data":{"outcome":{"conclusion":"TIMED_OUT","description":"Compilation timed out."},"persistentLogs":[{"name":"firstLog","uri":"http://myHost.com/firstLog"},{"name":"otherLog","uri":"isbn:0-486-27557-4"}]},"links":{"activityExecution":"aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeee1","flowContext":"flowContext","causes":["cause1","cause2"]},"meta":{"domainId":"example.domain","id":"9cdd0f68-df85-44b0-88bd-fc4163ac90a0","type":"eiffelactivityfinished","version":"0.1.7","time":1478780245184,"tags":["tag1","tag2"],"source":{"host":"host","name":"name","uri":"http://java.sun.com/j2se/1.3/","serializer":{"groupId":"G","artifactId":"A","version":"V"}}}}"]' http://localhost:8080/producer/msg?mp=eiffelsemantics
 
-[{"id":"9cdd0f68-df85-44b0-88bd-fc4163ac90a0","status_code":200,"result":"SUCCESS","message":"Event sent successfully"}]
+[{"id":"9cdd0f68-df85-44b0-88bd-fc4163ac90a0","statusCode":200,"result":"SUCCESS"}]
```
Comment (xmairaa) [inline index.html]: it is not statusCode, Its look like status_code

File: index.html [hunk 3]
```diff
@@ -182,181 +198,34 @@ <h2 id="project_tagline"></h2>
...
 $ curl -H "Content-Type: application/json" -X POST --data-binary "@test.data" http://localhost:8080/producer/msg?mp=eiffelsemantics
 
-[{"id":"9cdd0f68-df85-44b0-88bd-fc4163ac90a0","status_code":200,"result":"SUCCESS","message":"Event sent successfully"},{"id":"9cdd0f68-df85-44b0-88bd-fc4163ac90a1","status_code":200,"result":"SUCCESS","message":"Event sent successfully"}]
+[{"id":"9cdd0f68-df85-44b0-88bd-fc4163ac90a0","statusCode":200,"result":"SUCCESS"},{"id":"9cdd0f68-df85-44b0-88bd-fc4163ac90a1","statusCode":200,"result":"SUCCESS"}]
```
Comment (xmairaa) [inline index.html]: it is not statusCode, Its look like status_code

File: index.html [hunk 4]
```diff
@@ -182,181 +198,34 @@ <h2 id="project_tagline"></h2>
...
 </code></pre>
 
 <p><strong>Malformed input:</strong></p>
 
 <pre><code>$ curl -H "Content-Type: application/json" -X POST -d 'SOMETHING' http://localhost:8080/producer/msg?ud=fem001&mp=eiffel3
 
-[{"status_code":400,"result":"Bad Request","message":"Invalid event content, client need to fix problem in event before submitting again"}]
+{"timestamp":"May 31, 2016 9:14:17 AM","status":400,"error":"Bad Request","exception":"org.springframework.http.converter.HttpMessageNotReadableException","message":"Could not read JSON: Expected a com.google.gson.JsonArray but was com.google.gson.JsonPrimitive; nested exception is com.google.gson.JsonSyntaxException: Expected a com.google.gson.JsonArray but was com.google.gson.JsonPrimitive","path":"/producer/msg"}
```
Comment (xmairaa) [inline index.html]: Check once this format.

File: index.html [hunk 5]
```diff
@@ -182,181 +198,34 @@ <h2 id="project_tagline"></h2>
...
-	 "result":"Service Unavailable",
-	 "message":"Please check previous event and try again later"
-	},
-	{
-	 "status_code":503,
-	 "result":"Service Unavailable",
-	 "message":"Please check previous event and try again later"
-	},
-]</code></pre>
 <p>You can open the RabbitMQ's management console and find these messages in a queue.</p>
```
Comment (xmairaa) [inline index.html]: Some content missing from the above line.

File: index.html (hunk 1 above)
Comment (SantoshNC68) [inline index.html]: I also think we missed message in the result, check once

File: index.html (hunk 2 above)
Comment (SantoshNC68) [inline index.html]: Missing message in the output

File: index.html (hunk 3 above)
Comment (SantoshNC68) [inline index.html]: Missing message in the output

File: index.html (hunk 1 above)
Comment (SantoshNC68) [inline index.html]: please check what fields are missed, there are a lot of places where the message is missing

File: index.html [hunk 6]
```diff
@@ -96,17 +113,17 @@ <h2 id="project_tagline"></h2>
...
 <p><strong>Publish on a given host, given exchange, given domain and given user domain suffix:</strong></p>
 
-<pre><code> $ java -jar publish.war -f publishMessages.json -en rabbit -mb mb101 -domain eiffel001 -ud fem001 -mp eiffelsemantics
+<pre><code> $ java -jar remrem-publish-cli.jar -f publishMessages.json -en rabbit -mb mb101 -domain eiffel001 -ud fem001 -mp eiffelsemantics
 </code></pre>
 
 <p><strong>If you want to have the message non persistent add np flag:</strong></p>
 
-<pre><code> $ java -jar publish.war -f publishMessages.json -en rabbit -mb mb123 -np -mp eiffelsemantics
+<pre><code> $ java -jar remrem-publish-cli.jar -f publishMessages.json -en rabbit -mb mb123 -np -mp eiffelsemantics
```
Comment (SantoshNC68) [inline index.html]: I think the jar name mentioned here should be something like remrem-publish-cli-<version>.jar

File: index.html (hunk 5 above)
Comment (SantoshNC68) [inline index.html]: Why is this removed? Doesn't this reflect the changes made by us for adding StatusCodes into the document?
